assets/js/blocks/cart-checkout/payment-methods/saved-payment-method-options.js

```diff
@@ -114,12 +114,9 @@ const SavedPaymentMethodOptions = () => {
 
 	const updateToken = useCallback(
 		( token ) => {
-			if ( token === '0' ) {
-				setPaymentStatus().started();
-			}
 			setActiveSavedToken( token );
 		},
-		[ setActiveSavedToken, setPaymentStatus ]
+		[ setActiveSavedToken ]
```

Comment:
This logic hasn't been needed ever since we switched away from tabs and radios to only radios for payment method selection. There is no longer a possibility for a token with a value of `'0'` in the checkout flow.